src/test/java/apicurito/tests/utils/slenide/ConfigurationOCPUtils.java

```diff
@@ -0,0 +1,147 @@
+package apicurito.tests.utils.slenide;
+
+import static org.assertj.core.api.Assertions.fail;
+
+import org.yaml.snakeyaml.Yaml;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import apicurito.tests.configuration.TestConfiguration;
+import apicurito.tests.utils.HttpUtils;
+import apicurito.tests.utils.openshift.OpenShiftUtils;
+import io.fabric8.kubernetes.api.model.apps.ReplicaSet;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class ConfigurationOCPUtils {
+
+    public static void waitForOperatorUpdate() {
```

Comment:
Seems like in the two for loops to be some code repetition - could be made into separate method, to make it more readable.